palette/src/rgb/packed.rs

```diff
@@ -49,6 +49,22 @@ use crate::Pixel;
 /// When an `Rgb` type is packed, the alpha value will be `0xFF` in the
 /// corresponding `u32`. Converting from a packed color type back to an `Rgb`
 /// type will disregard the alpha value.
+///
+/// `Packed` implements [Pixel](encoding/pixel/trait.Pixel.html) and can be
+/// contructed from a slice of `&[u32]`.
```

Comment:
I just noticed a typo:

```suggestion
/// constructed from a slice of `&[u32]`.
```